src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommandRunner.cs

```diff
@@ -93,12 +93,12 @@ public async Task<int> ExecuteCommand(PackageReferenceArgs packageReferenceArgs,
             }
 
             // Parse the user specified frameworks once to avoid re-do's
-            var userSpecifiedFrameworks = Enumerable.Empty<NuGetFramework>();
+            var userSpecifiedFrameworks = new List<NuGetFramework>();
             if (packageReferenceArgs.Frameworks?.Any() == true)
             {
-                userSpecifiedFrameworks = packageReferenceArgs
+                userSpecifiedFrameworks.AddRange(packageReferenceArgs
```

Comment:
Nice - Is this mainly a perf enhancement ?

Reply:
Yes. I changed `GenerateLibraryDependency` to use `List<NuGetFramework>` instead of `IEnumerable<NuGetFramework>`. That means existing call sites can no longer use `Enumerable.Empty<NuGetFramework>` when there are none.